### PR TITLE
readme.md transformations and ported unit tests for AzureBridge module

### PR DESCRIPTION
**Pull request by ionuttulai**
## Description

Added the YAML part for replacements in module readme.md.
Ported and adapted unit tests for AzureRM module to the new format. - **Needs Validation**

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/StackAdmin/Azs.AzureBridge.Admin/readme.md [hunk 1]
```diff
@@ -57,16 +57,52 @@ csproj: Azs.AzureBridge.Admin.csproj
...
-      suppress-format: true
+      subject-prefix: AzureBridge
+  - where:
+      subject: DownloadedProduct
+    set:
+      subject-prefix: AzureBridge
+  - where:
+      subject: Product
+    set:
+      subject-prefix: AzureBridge
```
Comment (bganapa) [inline src/StackAdmin/Azs.AzureBridge.Admin/readme.md]: you should be able to add a module level subject-prefix, @rakku-ms can you pls provide a sample?

File: src/StackAdmin/Azs.AzureBridge.Admin/readme.md [hunk 2]
```diff
@@ -57,16 +57,52 @@ csproj: Azs.AzureBridge.Admin.csproj
...
+
+  # Add alias for ResourceGroup to ResourceGroupName
+  - where:
+      parameter-name: ResourceGroup
```
Comment (bganapa) [inline src/StackAdmin/Azs.AzureBridge.Admin/readme.md]: i think we can change the swagger spec directly renaming resourceGroup to resourceGroupName

File: src/StackAdmin/Azs.AzureBridge.Admin/test/Get-AzsAzureBridgeActivation.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,56 @@
+$TestRecordingFile = Join-Path $PSScriptRoot 'Get-AzsAzureBridgeActivation.Recording.json'
```
Comment (bganapa) [inline src/StackAdmin/Azs.AzureBridge.Admin/test/Get-AzsAzureBridgeActivation.Tests.ps1]: we dont really need a new file for each test, pls feel free to group the tests in to a single file, if it makes sense

File: src/StackAdmin/Azs.AzureBridge.Admin/readme.md [hunk 4]
```diff
@@ -57,16 +57,52 @@ csproj: Azs.AzureBridge.Admin.csproj
...
+    remove: true.
+  - where:
+      verb: New
+    remove: true.
+  - where:
+      verb: Remove
+      subject: Activation
+    remove: true.
+
 subject-prefix: ''
```
Comment (rakku-ms) [inline src/StackAdmin/Azs.AzureBridge.Admin/readme.md]: you can define module level subject-prefix here. 
subject-prefix: 'AzureBridge'

File: src/StackAdmin/Azs.AzureBridge.Admin/readme.md (hunk 1 above)
Comment (rakku-ms) [reply src/StackAdmin/Azs.AzureBridge.Admin/readme.md]: see my next comment - line 106

File: src/StackAdmin/Azs.AzureBridge.Admin/readme.md (hunk 2 above)
Comment (ionuttulai) [reply src/StackAdmin/Azs.AzureBridge.Admin/readme.md]: Changes were made in azure-rest-api-specs repo

File: src/StackAdmin/Azs.AzureBridge.Admin/test/Get-AzsAzureBridgeActivation.Tests.ps1 (hunk 3 above)
Comment (ionuttulai) [reply src/StackAdmin/Azs.AzureBridge.Admin/test/Get-AzsAzureBridgeActivation.Tests.ps1]: I could group the Product related tests in one file but, as validation goes, I found that it was easier to keep tests for each cmdlet separate

**Pull request merged**